martin-tile-utils/src/lib.rs

```diff
@@ -294,61 +315,75 @@ mod tests {
     }
 
     #[test]
-    fn test_tile_index() {
-        assert_eq!((0, 0), tile_index(-180.0, 85.0511, 0));
+    fn test_tile_colrow() {
+        assert_eq!((0, 0), tile_colrow(-180.0, 85.0511, 0));
     }
 
     #[test]
     fn test_xyz_to_bbox() {
```

Comment:
Due to the inherent precision limitations of floating point numbers, performing back-and-forth transformations between XYZ and bounding box coordinates within a single unit test  seems not a good idea? I removed the transform back test code. And actually if you add them back, you can see some test failed. Any idea? I can't see logic error in these methods. Is it our const not accurate. My mind get lost now.

Reply:
Or maybe we should reserve `tile-grid` in this?